src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private int count;
```

Comment:
I dont think we should store fruit count in Fruit class. We can create some another model smth like TransactionDto, that wil have fruit, operation and quantity. We can parse input file in List<TransactionDto> and perform operations on it